backend/src/data/repositories/podcast/podcast.repository.ts

```diff
@@ -33,8 +33,8 @@ class Podcast {
     return this.#PodcastModel.query().updateAndFetchById(id, payload);
   }
 
-  public getAllByUserId(userId: string): Promise<TPodcast[]> {
-    return this.#PodcastModel.query().where({ user_id: userId }).withGraphJoined('image');
+  public getAllByUserId(filterParams: { user_id: string }): Promise<TPodcast[]> {
```

Comment:
May be better don`t change getAllByUserId. This function has to return all users by id, and then to filter them  on level up in podcastService.

Reply:
if we have a large database, processing it with js will be very expensive. IMHO :-)